tests/removeTask.test.js

```diff
@@ -0,0 +1,42 @@
+// Import the necessary modules for the tests
+import Task from '../src/modules/taskList.js';
+import addTask from '../src/modules/addTask.js';
+
+// Test suite for the addTask function
+describe('addTask function', () => {
+  // Test case 1: Adding a new task to an empty array
+  it('should add a new task to an empty array', () => {
+    const tasks = [];
+    const taskInput = { value: 'Buy groceries' };
+
+    const result = addTask(taskInput, tasks);
+
+    // Assert: Check that the task was added to the array
+    expect(result).toEqual([new Task('Buy groceries', false, 1)]);
+  });
+
+  // Test case 2: Adding a new task to a non-empty array
+  it('should add a new task to a non-empty array', () => {
+    const tasks = [new Task('Buy groceries', false, 1)];
+    const taskInput = { value: 'Do laundry' };
+
+    const result = addTask(taskInput, tasks);
+
+    // Assert: Check that the task was added to the array
+    expect(result).toEqual([
+      new Task('Buy groceries', false, 1),
+      new Task('Do laundry', false, 2),
+    ]);
+  });
+
+  // Test case 3: Not adding a new task if the input is empty
+  it('should not add a new task if the input is empty', () => {
+    const tasks = [];
+    const taskInput = { value: '' };
+
+    const result = addTask(taskInput, tasks);
+
+    // Assert: Check that the array is still empty
+    expect(result).toEqual([]);
+  });
+});
```

Comment:
- [ ] You did a great job testing the add item functionality. :+1: This entire file seems to be a duplicate of `addTask.test.js` and does not hold any tests regarding the delete functionality. Kindly ensure your tests also cover the delete functionality. 🧐️

Reply:
Thanks a lot sir for your kind feedback. I have overlooked it. I am really happy that you gave me such nice suggestions to improve my code. I have updated it as you mentioned. Thanks

Reply:
This change has been done well